components/Checkbox/Checkbox.css

```diff
@@ -50,6 +50,10 @@
   background: var(--color-white);
 }
 
+.input:focus + .label:before {
+  box-shadow: 0 0 3px 1px var(--color-blue);
```

Comment:
More inclined to move the `FormInput` colour to blue, for cases where we haven't overridden the browser default. Both `Checkbox` and `Radio` have a heavy use of a primary colour and teal would be too similar also.